### ISSUE TITLE
FMS 2024.01

### ISSUE THREAD

**Issue by edwardhartnett** [NOAA-EMC/WCOSS2-requests#6]
Has been installed on prod.
Needs to be tested by UFS but is needed by RRFS.

**Comment by edwardhartnett** [comment]
Is this going to be tested by the UFS team?

**Comment by junwang-noaa** [comment]
Again, the library team can test it by running UFS weather model RT, and let the UFS WM  code manager if there is any issues/questions. Currently the UFS wm team is resolving the issue with ESMF 8.6.1 and MAPL 2.46.2, issue #5.

**Comment by edwardhartnett** [comment]
UFS develop branch passes three regression tests for @Hang-Lei-NOAA on cactus. The tests he ran are:
* ATM CPLD control P8 
* ATM Aerosol test
* ATM control P8

@BrianCurtis-NOAA is doing further testing with Hang's module files.

**Comment by edwardhartnett** [comment]
@Hang-Lei-NOAA and @BrianCurtis-NOAA any further progress on this?

**Comment by BrianCurtis-NOAA** [comment]
Apologies, forgot there was a issue here setup: from https://github.com/ufs-community/ufs-weather-model/issues/2358#issuecomment-2322714946 this can be moved to a final install location on WCOSS2.

**Comment by edwardhartnett** [comment]
@Hang-Lei-NOAA have you requested that this be copied to the final location on WCOSS2?

**Comment by Hang-Lei-NOAA** [comment]
I have requested. But NCO require to load as following:
module use
/apps/ops/prod/libs/modulefiles/mpi/intel/19.1.3.304/cray-mpich/8.1.9
module load fms/2024.01

So, that means every time you have to use the path under prod and then
loading.

**Comment by BrianCurtis-NOAA** [comment]
@Hang-Lei-NOAA So I understand, normally they'll move it to a more common location but they won't with this, so our modulefiles will need to be adjusted accordingly? Or are we still waiting on them to move it?

**Comment by Hang-Lei-NOAA** [comment]
They won't do further.
We have to accept this and adjust our model.

**Comment by BrianCurtis-NOAA** [comment]
OK, let me get a PR going with this location for UFSWM. I'll also bump `ip` and `w3emc` to match what's loaded by default on WCOSS2 (4.0.0 and 2.12.0 respectively). 2.12.0 is higher than default with spack-stack, but I don't know of any issues with newer versions of w3emc and the UFSWM, anyone here have knowledge of any?

**Comment by junwang-noaa** [comment]
@Hang-Lei-NOAA would you please provide an updated module (see: https://github.com/ufs-community/ufs-weather-model/blob/develop/modulefiles/ufs_wcoss2.intel.lua) that can be properly loaded in UFS weather model?

**Comment by BrianCurtis-NOAA** [comment]
@junwang-noaa this is a simple change IMO. I'll take care of this one.

**Comment by edwardhartnett** [comment]
I believe the NCEPLIBS team is done with this issue. Please let us know if that's not the case.

**Comment by BrianCurtis-NOAA** [comment]
Thanks @edwardhartnett! The changes are about to be merged into UFSWM to use the fruits of this work.
